README.md

```diff
@@ -0,0 +1,2 @@
+# `Mx`
+> declarative object manipulation
```

Comment:
I'd like to see a readme with at least the license and an example or description of what this is.